File: front-end/src/app/entity/MusicManageDTO.ts
export class MusicManageDTO {
  /**
   * 数据库主键ID
   */
  id: number;
  /**
   * 音乐ID
   */
  musicId: string;
  /**
   * 音乐名
   */
  name: string;
  /**
   * 歌手
   */
  singer: string;
  /**
   * 歌词ID
   */
  lyricId: string;
  /**
   * 音乐类型
   *
   */
  type: number;
  /**
   * 创建时间
   */
  gmtCreate: string;
  /**
   * 更新时间
   */
  gmtModified: string;
  /**
   * 歌曲URI
   */
  musicUri: string;
  /**
   * 歌词URI
   */
  lyricUri: string;
  /**
   * 封面URI
   */
  coverUri: string;
}

File: nas-music/src/main/java/top/itning/yunshunas/music/dto/MusicManageDTO.java
package top.itning.yunshunas.music.dto;

import lombok.Data;

import java.io.Serializable;
import java.net.URI;
import java.util.Date;

/**
 * 音乐信息 后台专用
 *
 * @author itning
 * @since 2022/2/3 9:41
 */
@Data
public class MusicManageDTO implements Serializable {
    /**
     * 数据库主键ID
     */
    private Long id;
    /**
     * 音乐ID
     */
    private String musicId;
    /**
     * 音乐名
     */
    private String name;
    /**
     * 歌手
     */
    private String singer;
    /**
     * 歌词ID
     */
    private String lyricId;
    /**
     * 音乐类型
     * {@link top.itning.yunshunas.music.constant.MusicType}
     */
    private Integer type;
    /**
     * 创建时间
     */
    private Date gmtCreate;
    /**
     * 更新时间
     */
    private Date gmtModified;
    /**
     * 歌曲URI
     */
    private URI musicUri;
    /**
     * 歌词URI
     */
    private URI lyricUri;
    /**
     * 封面URI
     */
    private URI coverUri;
}
